src/main/java/com/bloomtechlabs/coderheroesbea/controllers/ProfilesController.java

```diff
@@ -0,0 +1,64 @@
+package com.bloomtechlabs.coderheroesbea.controllers;
+
+import com.bloomtechlabs.coderheroesbea.entities.Profiles;
+import com.bloomtechlabs.coderheroesbea.services.ProfilesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/profile/")
```

Comment:
This is super minor, but I don't think you need the extra '/' at the end of "/profile" in the RequestMapping annotation.